scripts/ci/build.sh

```diff
@@ -12,8 +22,7 @@ else
     # Use last commit date rather than build date to avoid ending up with builds for
     # different platforms having different version strings (and therefore producing different bytecode)
     # if the CI is triggered just before midnight.
-    last_commit_timestamp=$(git log -1 --date=iso --format=%ad HEAD)
-    date -d "$last_commit_timestamp" -u "+ci.%Y.%-m.%-d" >prerelease.txt
+    last_commit_date >prerelease.txt
```

Comment:
```suggestion
    git log -1 --date="format:%Y.%-m.%-d" --format="+ci.%cd" HEAD >prerelease.txt
```
Shouldn't just this work everywhere?

Reply:
It uses local timezone so it will be off by a few hours, depending on how servers are configured.

But I have just stumbled upon a suggestion to use `TZ` variable and this actually works:
```bash
TZ=UTC git log -1 --date="format-local:%Y.%-m.%-d" --format="+ci.%cd" HEAD
```

I will need to find a way to export a variable in PowerShell but yeah, apart from that it's simpler and much more portable.

Reply:
Done.